components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/FirefoxAccount.kt

```diff
@@ -100,12 +100,26 @@ class FirefoxAccount(handle: FxaHandle, persistCallback: PersistCallback?) : Aut
      *
      * @param scopes List of OAuth scopes for which the client wants access
      * @param entrypoint to be used for metrics
+     * @param metricsParams optional parameters used for metrics
      * @return String that resolves to the flow URL when complete
      */
-    fun beginOAuthFlow(scopes: Array<String>, entrypoint: String): String {
+    fun beginOAuthFlow(
+        scopes: Array<String>,
+        entrypoint: String,
```

Comment:
Was thinking about making `entrypoint` a part of the `metricsParams`, but that doesn't enforce it. Should it be enforced? Otherwise, I'll just put it in the metrics params and simplify this

Reply:
I recall discussion with @vladikoff that we thought the `entrypoint` should be required, and hence not part of the metrics params.